### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by OpondoG**
Set up a Hello Microverse and configured linters for code formatting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <h1>Hello Microverse!</h1>
```
Comment (danmainah) [inline index.html]: - [x] Kindly move this code to your body tag, it's not supposed to be in the header.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @OpondoG , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used


# Required Changes ♻️
- [ ] Kindly ensure all your linter checks are passing, I can see your Lighthouse and Webhint are failing.

_Check the comments under the review._

## Optional suggestions

- [ ] Kindy ensure your commits message are in the correct tense and are meaningful n future commits eg instead of `Added a gitignore file` it should be `Add a .gitignore file`

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (OpondoG) [reply index.html]: okay thanks

File: index.html [hunk 2]
```diff
@@ -5,12 +5,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
```
Comment (Sahar-AbdelSamad) [inline index.html]: - [ ] [OPTIONAL] Great job adding the `h1` tag to the body 👏  👍 , but you don't need to delete this line.
This is the link to the external style sheet `style.css` we need it to have the color of `Hello Microverse!` green like before.
Please add it back.

**Review by Sahar-AbdelSamad — APPROVED** [review]
## Status Approved 🎉 🥳 

Hi @OpondoG , :wave::wave:

Great work on making the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✔️ Linters are passing
✔️ Correct use of GitHub flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**